tests/casework/test_enrich_slug.py

```
@@ -0,0 +1,231 @@
+    assert record["slug_proposed"] == "sunil-poudel-land-fraud-080-cr-0047"
+    assert record["patch_ops"] == [
+        {
+            "op": "replace",
+            "path": "/slug",
+            "value": "sunil-poudel-land-fraud-080-cr-0047",
+        }
+    ]
+
+
+def test_apply_patches_valid_slug():
+    api, stats, proposals = _run(
```

Ruff (0.15.17)

[warning] Unpacked variable `proposals` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)